guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -64,19 +53,10 @@ If NFS share is already mounted, remount it using the above configuration and en
 
 Packages that are duplicated in different repositories are only stored once on the disk.
 Additional repositories containing duplicate packages require less additional storage.
-The bulk of storage resides in the `/var/lib/mongodb/` and `/var/lib/pulp/` directories.
+The bulk of storage resides in the `/var/lib/pulp/` directory.
 These end points are not manually configurable.
 Ensure that storage is available on the `/var` file system to prevent storage problems.
 
-.Temporary Storage
```

Comment:
Note to any reviewer: I am working to get guidance from Pulp team on changes in this file.

Reply:
Per https://github.com/theforeman/puppet-pulpcore#installation-layout we do have `/var/lib/pulp/tmp` that is [WORKING_DIRECTORY](https://docs.pulpproject.org/pulpcore/settings.html#working-directory). That states:
> The directory used by workers to stage files temporarily. This defaults to /var/lib/pulp/tmp/.
Then it goes on:
> It is recommended that WORKING_DIRECTORY and MEDIA_ROOT exist on the same storage volume for performance reasons. Files are commonly staged in the WORKING_DIRECTORY and validated before being moved to their permanent home in MEDIA_ROOT.

In our case `MEDIA_ROOT` is `/var/lib/pulp/media`.